### ISSUE TITLE
sniff support plan

### ISSUE THREAD

**Issue by winterland1989** [bitemyapp/bloodhound#118]
Does bloodhound support cluster sniffing? if not,  is there any plan for cluster sniffing? This is a quiet important feature since my company's es cluster may add nodes in future.

**Comment by bitemyapp** [comment]
I wasn't aware cluster sniffing was available outside of the transport API. Without that, it's equivalent to polling the cluster metadata is it not?

If you want it, you're likely going to have to PR it.

**Comment by winterland1989** [comment]
Yes, basically it's a polling mechanism, my time is very limiting on this job now, so i'm writing the es client in go using [elastic v3](https://github.com/olivere/elastic) instead and talk to main haskell service using http endpoint, after this project is finish, i'll revise this part and see what i can do. thanks for your reply!

**Comment by winterland1989** [comment]
done in #119 , please review : )
